src/test/directives/class-map_test.ts

```diff
@@ -81,6 +81,16 @@ suite('classMap', () => {
     assert.isFalse(el.classList.contains('foo'));
   });
 
+  test('can override static classes', () => {
+    renderClassMapStatic({aa: false, bb: true});
+    const el = container.firstElementChild!;
+    assert.isFalse(el.classList.contains('aa'));
+    assert.isTrue(el.classList.contains('bb'));
+    renderClassMapStatic({});
+    assert.isFalse(el.classList.contains('aa'));
```

Comment:
This seems bad to me, but I guess it's current behavior. I'd much rather have the behavior that the output is the same for the same input, regardless of order of calls (with the exception of out-of-band classList mutations).

Reply:
Agreed. Wanna cut a v2 anytime soon?

Reply:
Not particularly :)

I'd want to move further on SSR and add a whole lot of benchmarks before that at least.